sample-compose-fragments/src/main/res/navigation/nav_graph.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<navigation xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/nav_graph"
+    app:startDestination="@+id/organismsFragment">
+
+    <fragment
+        android:id="@+id/organismsFragment"
+        android:name="uk.gov.hmrc.sample_compose_fragments.presentation.fragment.OrganismsFragment"
+        android:label="fragment_organisms"
```

Comment:
In the non-compose compatible navigation graphs, the `label` attribute is shown as the toolbar title. Is that the same here? If so, can we name it something more user friendly: "Organisms". Same for below ("Molecules", "Atoms", "Colours")